_data/retail.yml

```
@@ -301,18 +355,25 @@ websites:
     img: nordstrom.png
     twitter: Nordstrom
     facebook: Nordstrom
+    regions:
+      - us
 
   - name: Northern Tool + Equipment
     url: https://www.northerntool.com/
     img: northerntool.png
     twitter: NorthernTool
     facebook: northerntool
+    regions:
+      - us
 
   - name: Office Depot
     url: https://www.officedepot.com
     twitter: officedepot
     facebook: OfficeDepot
     img: officedepot.png
+    regions:
+      - ca
+      - us
```

```suggestion
    regions:
      - us
```
The Office Depot website specifically serves the US. Canada's version of Office Depot is called [Grand & Toy](https://www.grandandtoy.com/).